### PR TITLE
Get started

### PR DESCRIPTION
**Pull request by JackHMRC**

### CODE REVIEW

**Review by Jon-Rowe-HMRC — CHANGES_REQUESTED** [review]
@timsb can the homepage button be styled like https://design-system.service.gov.uk 'Get started >' but switching their blue for our green.

Think the homepage should be clear that our patterns are supplemental. Maybe a new first line in the principles sections. Something like 'HMRC design resources supplements the [GOV.UK Design System](https://design-system.service.gov.uk). It follows the....'

I'll hold on comments on get started for now until you've changed the dir to /get-started/ and previewed it yourself (there's a few styling issues).

**Comment by Jon-Rowe-HMRC** [comment]
In addition to the previous points...

'(Contractors/consultants)' > '(contractors/consultants)'?

'Mural' is the expectation for everyone to have an account? I thought Ops were still trying to restrict to people who really want it. I might be out of date. If not, remove from 'What you will need' as it's just guest access that anyone has?

'#team-design-system' add the link https://hmrcdigital.slack.com/archives/C014S77JCJW?name=C014S77JCJW#

'Service patterns also have a link to the team’s discussions on [GitHub](https://github.com/hmrc/design-patterns/issues).' > 'Service patterns also link to a [GitHub](https://github.com/hmrc/design-patterns/issues) discussion.'? (much of the discussion isn't the team)

'#gov uk-design-system' > '#govuk-design-system' and add link.

'HMRC’s preferred approach is to design using low-fidelity prototypes in the early stages of your project.' > 'HMRC’s preferred approach is to design using low-fidelity prototypes in early project stages.'

change any en dashes to em dashes

'The following tools are not authorised for use right now:' > 'The following tools are not authorised for use by service teams:'? Right now feels like false promise, and Figma/Sketch were authorised for the app team.

'"class=' > '" class=' (lots of these on what's new, do a mass search and replace)

Overall, looks really good. Lots of positive changes and helpful new content.

**Comment by JackHMRC** [comment]
Hi @Jon-Rowe-HMRC 

I think I've covered all your points apart from the "'"class=' > '" class='" issue you raised. Tbh, I'm not sure what the issue is with this one, could you expand a bit on what needs doing please?

Cheers

Jack

**Comment by Jon-Rowe-HMRC** [comment]
> Hi @Jon-Rowe-HMRC
> 
> I think I've covered all your points apart from the "'"class=' > '" class='" issue you raised. Tbh, I'm not sure what the issue is with this one, could you expand a bit on what needs doing please?
> 
> Cheers
> 
> Jack

It's missing spaces before 'class'. For example:
<a href="https://www.gov.uk/guidance/government-design-principles"class="govuk-link">

Should be:
<a href="https://www.gov.uk/guidance/government-design-principles" class="govuk-link">

**Comment by JackHMRC** [comment]
Cheers Jon. These changes have been made.

**Review by Jon-Rowe-HMRC — APPROVED** [review]

**Comment by Jon-Rowe-HMRC** [comment]
Not stopping publishing but on...
'HMRC design resources supplement GOV.UK Design System.'
The full stop shouldn't be part of the link. Should there be a 'the' before GOV.UK?  Feel free to ignore if you disagree :)

**Comment by Jon-Rowe-HMRC** [comment]
'#gov uk-prototype-kit' > '#govuk-prototype-kit'

There's still some links with no space before the class eg '"class' should be '" class'

**Review by Jon-Rowe-HMRC — APPROVED** [review]

**Comment by Jon-Rowe-HMRC** [comment]
'automatic prototype creation tool' has been changed to all straight quotes, but they should be curled

File: src/get-started/index.njk [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+<p class="govuk-body">
+If you need access to items on this list, speak to your manager/project lead, who will contact P&UCD Ops to set you up.
+</p>
+
+<h2 class="govuk-heading-l" id="Patterns-components-and-styles">
+  Patterns, components and styles
+</h2>
+
+<p class="govuk-body">
+All designs for HMRC services must follow <a href="https://www.gov.uk/guidance/government-design-principles" class="govuk-link">GOV.UK principles</a> and use <a href="https://design.tax.service.gov.uk/hmrc-design-patterns/" class="govuk-link">HMRC</a> and <a href="https://www.gov.uk/guidance/government-design-principles" class="govuk-link">GOV.UK</a>
```
Comment (timsb) [inline src/get-started/index.njk:52]: The links 'HMRC' & 'GOV.UK' don't go to where you'd expect them to out of context. Could you change them to be something like 'HMRC design patterns' & 'GOV.UK design principles'?

File: src/get-started/index.njk [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+<h3 class="govuk-heading-m" id="Get-support">
+  Get support
+</h3>
+
+<p class="govuk-body">
+Designers can get support with HMRC-specific patterns from the HMRC Design Resources team on HMRC Slack at <a href="https://hmrcdigital.slack.com/archives/C014S77JCJW?name=C014S77JCJW#" class="govuk-link">#team-design-system</a>. Service patterns also have a link to the discussions on <a href="https://github.com/hmrc/design-patterns/issues" class="govuk-link">GitHub</a>.
+</p>
+
+<p class="govuk-body">
+For wider government patterns and components, the GOV.UK Design System Team is available on cross-government Slack at <a href="https://hmrcdigital.slack.com/archives/C014S77JCJW?name=C014S77JCJW#">#team-design-system</a>.
```
Comment (timsb) [inline src/get-started/index.njk:73]: Not sure this is the correct link for the cross GOV slack channel?

Comment (timsb) [reply src/get-started/index.njk:73]: Doesn't make sense out of context. Could you include more of the copy to be part of the link e.g. 'cross-government Slack at #team-design-system'?

File: src/get-started/index.njk [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+<p class="govuk-body">
+Designers should use these libraries for internal services where possible. When prototyping services for internal users, or services not on GOV.UK, designers should follow the relevant brand and typography guidance for their platform. 
+</p>
+
+<h3 class="govuk-heading-m" id="Get-support">
+  Get support
+</h3>
+
+<p class="govuk-body">
+Designers can get support with HMRC-specific patterns from the HMRC Design Resources team on HMRC Slack at <a href="https://hmrcdigital.slack.com/archives/C014S77JCJW?name=C014S77JCJW#" class="govuk-link">#team-design-system</a>. Service patterns also have a link to the discussions on <a href="https://github.com/hmrc/design-patterns/issues" class="govuk-link">GitHub</a>.
```
Comment (timsb) [inline src/get-started/index.njk:69]: These links don't make sense out of context. Could you change them to something like 'HMRC Slack at #team-design-system' & 'HMRC Github issues'?

File: src/get-started/index.njk [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+<h2 class="govuk-heading-l" id="Low-fidelity-prototypes">
+Low-fidelity prototypes
+</h2>
+
+<p class="govuk-body">
+HMRC’s preferred approach is to design using low-fidelity prototypes in early project stages.
+</p>
+    
+<p class="govuk-body">
+Designers can use low-fidelity prototypes to explore ideas quickly and cheaply before creating ‘high-fidelity’ prototypes that look more like the final product. Low-fidelity prototypes should be used in addition to high-fidelity prototypes - not instead of them.
```
Comment (timsb) [inline src/get-started/index.njk:101]: There's a mixture of cases 'Low-fidelity' & 'low-fidelity'. Is on of them correct or are they both correct depending on how you're using them?

File: src/get-started/index.njk [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+     
+<p class="govuk-body">
+Designers should help teams and stakeholders understand how low-fidelity prototypes help projects quickly and cheaply explore design ideas.
+</p>
+
+<h3 class="govuk-heading-m" id="Low-fidelity-prototyping-tools">
+Low-fidelity prototyping tools
+</h3>
+
+<h3 class="govuk-heading-s" id="sketches">
```
Comment (timsb) [inline src/get-started/index.njk:112]: I think this should be an `h4`

File: src/get-started/index.njk [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+
+<p class="govuk-body">
+Sketched prototypes, using pen and paper, are the easiest way to start exploring design ideas.
+</p>
+
+<p class="govuk-body">
+Sketches are cheap and easy to produce and can be discarded if they don’t meet your needs. This makes them perfect for the early stages of designing a service or product.
+</p>
+
+<h3 class="govuk-heading-s" id="google-suite">
```
Comment (timsb) [inline src/get-started/index.njk:124]: I think this should be an `h4`

File: src/get-started/index.njk [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
+Google Suite
+</h3>
+
+<p class="govuk-body">
+Designers with access to Google Suite have a range of tools available for creating journey maps, flow diagrams, quick sketches, and forms. 
+</p>
+
+<h3 class="govuk-heading-s" id="mural">
```
Comment (timsb) [inline src/get-started/index.njk:132]: I think this should be an `h4`

File: src/get-started/index.njk [hunk 8]
```diff
@@ -0,0 +1,237 @@
...
+Mural
+</h3>
+
+<p class="govuk-body">
+Mural is available for use on HMRC devices and we recommend using it for creating flow diagrams, journey maps, sketching out design ideas and creating simple screens. 
+</p>
+
+<p class="govuk-body">
+Full licences may not be available for every designer, please speak to the design leadership in your programme or P&UCD Ops to find out what is available for you.
```
Comment (timsb) [inline src/get-started/index.njk:141]:
```suggestion
Full licences may not be available for every designer, please speak to the design leadership in your programme or P&amp;UCD Ops to find out what is available for you.
```

File: src/get-started/index.njk [hunk 9]
```diff
@@ -0,0 +1,237 @@
...
+</p>
+
+<p class="govuk-body">
+  Mural templates for low-fidelity prototyping:
+  
+  <ul class="govuk-list govuk-list--bullet">
+    <li><a href="https://github.com/dashouse/govuk-design-system-flow-diagrams" class="govuk-link">Flow diagrams for GOV.UK services</a></li>
+    <li><a href="https://github.com/clare-brown/govuk-designsystem-flow-diagram-mural" class="govuk-link">GOV.UK elements in Mural</a></li>
+    </ul>
+    </p>
```
Comment (timsb) [inline src/get-started/index.njk:151]: The `<p>` should close off before the `<ul>` starts

File: src/get-started/index.njk [hunk 10]
```diff
@@ -0,0 +1,237 @@
...
+
+<h2 class="govuk-heading-l" id="High-fidelity-prototypes">
+ High-fidelity prototypes
+ </h2>
+ 
+ <p class="govuk-body">
+ High-fidelity prototypes are detailed, coded prototypes that mimic the behaviour of ‘finished’ services. High-fidelity prototypes will help you to perform user testing and demonstrate the service to stakeholders.
```
Comment (timsb) [inline src/get-started/index.njk:158]: For the second 'High-fidelity' should it be lower case?

File: src/get-started/index.njk [hunk 11]
```diff
@@ -0,0 +1,237 @@
...
+ <h3 class="govuk-heading-m" id="Prototype-support">
+ Prototype support
+ </h3>
+ 
+ <p class="govuk-body">
+ The GOV.UK Prototype Kit is used across governments and there is a significant amount of community support available if you get stuck or need help. 
+ </p>
+ 
+ <p class="govuk-body">
+ Inside HMRC you can reach out in our #community-prototype Slack channel or speak to members of HMRC’s Design Resources Team. Support is also available on the cross-government Slack in the #govuk-prototype-kit channel.
```
Comment (timsb) [inline src/get-started/index.njk:178]: We've linked to Slack channels above. Should we be doing the same for these ones?

File: src/get-started/index.njk [hunk 12]
```diff
@@ -0,0 +1,237 @@
...
+ <h3 class="govuk-heading-m" id="Hosting-and-sharing-prototypes">
+ Hosting and sharing prototypes
+ </h3>
+ 
+ <p class="govuk-body">
+ You will need a GitHub account, which you can create, and a digital HMRC login which can be set up for you by your team lead. Once you have set up your account you can use the 'automatic prototype creation tool'. 
+ </p>
+
+ <p class="govuk-body">
+ If you are working on a new service or a new prototype follow the guidance on <a href="https://confluence.tools.tax.service.gov.uk/display/PROT/Creating+a+new+prototype+repository/" class="govuk-link">Confluence</a> to create a new prototype repository. If you are working on a service where a prototype already exists, speak to the senior interaction designer on your programme or reach out on HMRC Slack <a href="https://join.slack.com/share/enQtNzY0NjE3NjY5NDE4MS1lNDczNTQxODk5YzE4MDk0MGY5YTBkY2YwMDdiNWNmZDRhMDdkOGIxMDg3YjA0ZTg5OWUwYjZhMTNjNjFlNjc3" class="govuk-link">#community-prototype</a>.
```
Comment (timsb) [inline src/get-started/index.njk:194]: I don't think these links don't go where I'd expect them to go out of context. For '#community-prototype' could the link be expanded out to 'HMRC Slack #community-prototype'?

File: src/get-started/index.njk [hunk 13]
```diff
@@ -0,0 +1,237 @@
...
+ <h2 class="govuk-heading-l" id="Other-tools-to-support-design-and-prototyping">
+ Other tools to support design and prototyping
+ </h2>
+ 
+ <h3 class="govuk-heading-m" id="Prototype-support">
+ Pattern Checker
+ </h3>
+ 
+ <p class="govuk-body">
+ Pattern Checker is an <a href="https://chromewebstore.google.com/detail/pattern-checker/amjjliajblignodfdjalnfkekkeflkph?hl=en-GB">automated pattern and accessibility checker</a> for government services that use GOV.UK and HMRC design patterns.
```
Comment (timsb) [inline src/get-started/index.njk:214]: Not everyone uses Chrome. Should we mentioned that this is a Chrome browser extension?

File: src/get-started/index.njk [hunk 14]
```diff
@@ -0,0 +1,237 @@
...
+ <li>Sketch</li>
+ </ul>
+ 
+ <p class="govuk-body">
+ If you have a question about any of these tools please speak to the head of your profession.
+ </p>
+
+
+
+
```
Comment (timsb) [inline src/get-started/index.njk:237]: Not a massive one but there's a number of blank lines at the end of this file. There should just be 1 blank line at the end of the file.

File: src/hmrc-content-style-guide/index.njk [hunk 15]
```diff
@@ -22,8 +22,10 @@ layout: oneColumnPage.njk
     </p>
 
     <p class="govuk-body">
-      Use the same terms, style and spelling and grammar conventions in GOV.UK content and services to make
-      the user’s end-to-end journey consistent.
+      You can also refer to the GOV.UK guidance on
+      <a href="https://www.gov.uk/guidance/content-design" class="govuk-link">planning, writing and managing content</a>
+      and
+<a href="https://www.gov.uk/service-manual/design/writing-for-user-interfaces" class="govuk-link">writing for user interfaces.</a>
```
Comment (timsb) [inline src/hmrc-content-style-guide/index.njk:28]: The full stop shouldn't be part of the link
```suggestion
<a href="https://www.gov.uk/service-manual/design/writing-for-user-interfaces" class="govuk-link">writing for user interfaces</a>.
```

File: src/index.njk [hunk 16]
```diff
@@ -16,72 +16,26 @@ layout: homepageLayout.njk
 
           <div class="govuk-grid-row">
             <div class="govuk-grid-column-one-third">
-              <a href="https://design-system.service.gov.uk" class="app-card">
-                <h3 class="govuk-heading-s app-card__heading">GOV.UK Design System</h3>
-                <p class="govuk-body">Make your service consistent with GOV.UK using approved styles, components and patterns.</p>
+              <a href=" hmrc-design-patterns/" class="app-card">
```
Comment (timsb) [inline src/index.njk:19]: Although this works as the browser has worked out what you meant it should be:
```suggestion
              <a href="/hmrc-design-patterns/" class="app-card">
```

File: src/index.njk [hunk 17]
```diff
@@ -16,72 +16,26 @@ layout: homepageLayout.njk
...
+                <p class="govuk-body">Design HMRC services with reusable, accessible patterns based on the research and experience of other service teams.
+</p>
               </a>
             </div>
 
             <div class="govuk-grid-column-one-third">
-              <a href="/hmrc-design-patterns" class="app-card">
-                <h3 class="govuk-heading-s app-card__heading">HMRC design patterns</h3>
-                <p class="govuk-body">Find patterns and guidance developed specifically for HMRC.</p>
+              <a href=" patterns-hmrc-govuk-team/" class="app-card">
```
Comment (timsb) [inline src/index.njk:27]: Although this works as the browser has worked out what you meant it should be:
```suggestion
              <a href="/patterns-hmrc-govuk-team/" class="app-card">
```

File: src/index.njk [hunk 18]
```diff
@@ -16,72 +16,26 @@ layout: homepageLayout.njk
...
-              <a href="https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style" class="app-card">
-                <h3 class="govuk-heading-s app-card__heading">GOV.UK content style guide</h3>
-                <p class="govuk-body">Use the same style, spelling and grammar conventions as GOV.UK.</p>
-              </a>
-            </div>
-
-            <div class="govuk-grid-column-one-third">
-              <a href="/hmrc-content-style-guide" class="app-card">
-                <h3 class="govuk-heading-s app-card__heading">HMRC content style guide</h3>
+              <a href=" hmrc-content-style-guide/" class="app-card">
```
Comment (timsb) [inline src/index.njk:34]: Although this works as the browser has worked out what you meant it should be:
```suggestion
              <a href="/hmrc-content-style-guide/" class="app-card">
```

File: src/index.njk [hunk 19]
```diff
@@ -90,60 +44,27 @@ layout: homepageLayout.njk
...
-            <div class="govuk-grid-column-one-third">
-              <a href="https://www.gov.uk/service-manual/helping-people-to-use-your-service" class="app-card">
-                <h3 class="govuk-heading-s app-card__heading">Accessibility and assisted digital</h3>
-                <p class="govuk-body">Help and encourage all users to use HMRC services.</p>
-              </a>
-            </div>
-          </div>
-        </div>
-      </div>
+           <p class="govuk-body">
```
Comment (timsb) [inline src/index.njk:49]: Looks like you've got an opening paragraph element where there shouldn't be one. Should be remove
```suggestion
```

File: src/index.njk [hunk 20]
```diff
@@ -90,60 +44,27 @@ layout: homepageLayout.njk
...
+      <p class="govuk-body"> HMRC design resources supplement <a href="https://design-system.service.gov.uk/">GOV.UK Design System.</a> It follows the GOV.UK <a href="https://www.gov.uk/guidance/government-design-principles" class="govuk-link">Government Design Principles</a>, <a href="https://www.gov.uk/service-manual/service-standard" class="govuk-link">Service Standard</a> and <a href="https://www.gov.uk/guidance/content-design" class="govuk-link">content design guidance</a>. HMRC also follows the GOV.UK <a href="https://design-system.service.gov.uk/accessibility/accessibility-strategy/" class="govuk-link">accessibility strategy</a> to improve the accessibility of our services.
```
Comment (timsb) [inline src/index.njk:50]: There's a full stop in a link. Should be after:
```suggestion
      <p class="govuk-body"> HMRC design resources supplement <a href="https://design-system.service.gov.uk/">GOV.UK Design System</a>. It follows the GOV.UK <a href="https://www.gov.uk/guidance/government-design-principles" class="govuk-link">Government Design Principles</a>, <a href="https://www.gov.uk/service-manual/service-standard" class="govuk-link">Service Standard</a> and <a href="https://www.gov.uk/guidance/content-design" class="govuk-link">content design guidance</a>. HMRC also follows the GOV.UK <a href="https://design-system.service.gov.uk/accessibility/accessibility-strategy/" class="govuk-link">accessibility strategy</a> to improve the accessibility of our services.
```

File: src/get-started/index.njk (hunk 2 above)
Comment (timsb) [reply src/get-started/index.njk:73]: Link needs `class='govuk-link'` added.

File: src/get-started/index.njk [hunk 21]
```diff
@@ -0,0 +1,237 @@
...
+ <p class="govuk-body">
+ High-fidelity prototypes are detailed, coded prototypes that mimic the behaviour of ‘finished’ services. High-fidelity prototypes will help you to perform user testing and demonstrate the service to stakeholders.
+ </p>
+ 
+ <p class="govuk-body">
+ HMRC’s preferred approach is to use the GOV.UK Prototype Kit. You will need a device, such as a MacBook, that isn’t managed by HMRC so you can install the necessary software. 
+ </p>
+ 
+ <p class="govuk-body">
+ You can <a href="https://design-system.service.gov.uk/get-started/prototyping/">follow instructions</a> in the GOV.UK Design System to get the kit running on your unmanaged device.
```
Comment (timsb) [inline src/get-started/index.njk:166]: Link needs `class='govuk-link`.

File: src/get-started/index.njk (hunk 13 above)
Comment (timsb) [reply src/get-started/index.njk:214]: Link needs `class='govuk-link'`.

File: application/templates/partials/homepageBanner.njk [hunk 22]
```diff
@@ -2,8 +2,16 @@
...
-        <h1 class="govuk-heading-xl hmrc-masthead__title">Use the HMRC design resources</h1>
-        <p class="hmrc-masthead__description">Use these design resources to make HMRC services consistent with GOV.UK. Employ GOV.UK and HMRC-specific patterns to learn from the experience of other teams and avoid repeating work.</p>
+        <h1 class="govuk-heading-xl hmrc-masthead__title">Design your service</h1>
+        <p class="hmrc-masthead__description">Use these design resources to make your service and guidance consistent with HMRC and GOV.UK. Learn from the experience of other teams to avoid repeating work and share your knowledge by suggesting new content.</p>
+        </p>
+        {% from "govuk/components/button/macro.njk" import govukButton %}
+
+{{ govukButton({
+  text: "Start now",
+  href: "http://localhost:3000/Get-started/",
```
Comment (timsb) [inline application/templates/partials/homepageBanner.njk:12]:
```suggestion
  href: "http://localhost:3000/get-started/",
```

**Review by timsb — CHANGES_REQUESTED** [review]

File: application/templates/partials/homepageBanner.njk [hunk 23]
```diff
@@ -2,8 +2,15 @@
...
       <div class="govuk-grid-column-three-quarters-from-desktop">
-        <h1 class="govuk-heading-xl hmrc-masthead__title">Use the HMRC design resources</h1>
-        <p class="hmrc-masthead__description">Use these design resources to make HMRC services consistent with GOV.UK. Employ GOV.UK and HMRC-specific patterns to learn from the experience of other teams and avoid repeating work.</p>
+        <h1 class="govuk-heading-xl hmrc-masthead__title">Design your service</h1>
+        <p class="hmrc-masthead__description">Use these design resources to make your service and guidance consistent with HMRC and GOV.UK. Learn from the experience of other teams to avoid repeating work and share your knowledge by suggesting new content.</p>
+        {% from "govuk/components/button/macro.njk" import govukButton %}
+
+{{ govukButton({
+  text: "Start now",
+  href: "http://localhost:3000/get-started/",
```
Comment (timsb) [inline application/templates/partials/homepageBanner.njk:11]:
```suggestion
  href: "/get-started/",
```

File: src/get-started/index.njk [hunk 24]
```diff
@@ -0,0 +1,236 @@
...
+Mural is available for use on HMRC devices and we recommend using it for creating flow diagrams, journey maps, sketching out design ideas and creating simple screens. 
+</p>
+
+<p class="govuk-body">
+Full licences may not be available for every designer, please speak to the design leadership in your programme or P&amp;UCD Ops to find out what is available for you.
+</p>
+
+<p class="govuk-body">
+  Mural templates for low-fidelity prototyping:
+
```
Comment (timsb) [inline src/get-started/index.njk:145]:
```suggestion
  </p>
```

File: src/get-started/index.njk [hunk 25]
```diff
@@ -0,0 +1,236 @@
...
+</p>
+
+<p class="govuk-body">
+  Mural templates for low-fidelity prototyping:
+  
+  <ul class="govuk-list govuk-list--bullet">
+    <li><a href="https://github.com/dashouse/govuk-design-system-flow-diagrams" class="govuk-link">Flow diagrams for GOV.UK services</a></li>
+    <li><a href="https://github.com/clare-brown/govuk-designsystem-flow-diagram-mural" class="govuk-link">GOV.UK elements in Mural</a></li>
+    </ul>
+    </p>
```
Comment (timsb) [inline src/get-started/index.njk:150]:
```suggestion
```

**Review by timsb — APPROVED** [review]

**Pull request merged**